core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5276,6 +5276,10 @@ components:
         monitoringImageScript:
           type: string
           description: When using `monitoringImage`, a user-defined script that runs inside the monitoring container
+        perWorkflowCostCap:
+          type: number
+          description: The maximum cost for a single workflow in this submission.  If the workflow exceeds this cost, it will be aborted.
+          format: float
```

Comment:
add info/validation that matches what the backend does, here: https://github.com/broadinstitute/rawls/blob/822aabeecdd74f9b88fb715b8bc5a739c28d766d/core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala#L569-L589 so users can predict what to enter